cmd/podman/mount.go

```diff
@@ -68,23 +69,37 @@ func mountCmd(c *cli.Context) error {
 		return errors.Errorf("%q is not a supported format", c.String("format"))
 	}
 
-	if len(args) > 1 {
-		return errors.Errorf("too many arguments specified")
-	}
-
-	if len(args) == 1 {
-		if json {
-			return errors.Wrapf(err, "json option can not be used with a container id")
-		}
-		ctr, err := runtime.LookupContainer(args[0])
-		if err != nil {
-			return errors.Wrapf(err, "error looking up container %q", args[0])
+	var lastError error
+	if len(args) > 0 {
+		for _, name := range args {
+			if json {
+				if lastError != nil {
+					logrus.Error(lastError)
+				}
+				lastError = errors.Wrapf(err, "json option can not be used with a container id")
+				continue
+			}
+			ctr, err := runtime.LookupContainer(name)
+			if err != nil {
+				if lastError != nil {
+					logrus.Error(lastError)
+				}
+				lastError = errors.Wrapf(err, "error looking up container %q", name)
+				continue
+			}
+			mountPoint, err := ctr.Mount()
+			if err != nil {
+				if lastError != nil {
+					logrus.Error(lastError)
+				}
+				lastError = errors.Wrapf(err, "error mounting container %q", ctr.ID())
+				continue
+			}
+			fmt.Printf("%s\n", mountPoint)
 		}
-		mountPoint, err := ctr.Mount()
-		if err != nil {
-			return errors.Wrapf(err, "error mounting container %q", ctr.ID())
+		if lastError != nil {
+			logrus.Error(lastError)
```

Comment:
I don't think we need to bother with this - just return lastError

Reply:
That does work and looks a little better output wise, thx for the suggestion.